nncf/quantization/advanced_parameters.py

```diff
@@ -283,46 +284,52 @@ def convert_range_estimator_parameters_to_dict(params: RangeEstimatorParameters)
             "min_percentile": 1 - params.min.quantile_outlier_prob,
             "max_percentile": 1 - params.max.quantile_outlier_prob,
         }
+    elif (
+        params.min.statistics_type is None
+        and params.min.aggregator_type is None
+        and params.max.statistics_type is None
+        and params.max.aggregator_type is None
+    ):
+        return {}
     else:
         raise RuntimeError("The following range estimator parameters are not supported: " f"{str(params)}")
 
     return result
 
 
-def convert_advanced_parameters_to_dict(params: AdvancedQuantizationParameters) -> Dict[str, Any]:
+def apply_advanced_parameters_to_config(config: NNCFConfig, params: AdvancedQuantizationParameters) -> NNCFConfig:
```

Comment:
`NNCFConfig` will be deprecated in the future. Could you prove the necessity of these changes?

Reply:
Removed from this PR, https://github.com/openvinotoolkit/nncf/pull/1906